internal/duration/duration.go

```diff
@@ -10,18 +10,21 @@ import (
 )
 
 const (
-	systemDomain        = 2 * time.Minute
-	appReady            = 2 * time.Minute
-	deployment          = 5 * time.Minute
-	orgDeletion         = 5 * time.Minute
-	serviceSecret       = 5 * time.Minute
-	serviceProvision    = 5 * time.Minute
-	serviceLoadBalancer = 5 * time.Minute
-	podReady            = 5 * time.Minute
-	appBuilt            = 10 * time.Minute
-	warmupJobReady      = 30 * time.Minute
-	certManagerReady    = 5 * time.Minute
-	kubedReady          = 5 * time.Minute
+	systemDomain             = 2 * time.Minute
```

Comment:
This seems weird to me. Maybe we should have `const slow = 2 * time.Minute`, `const medium`, `const long` instead of all these numbers?

Anyhow, not important.

Reply:
It feels weird to me too. We just gave names to magic numbers and now we have magic constants! In general, these are upper limits on the time we expect it to take. When that proves to be small, we just bump it until it's good. The reason for having multiple of them here is to avoid bumping the timeout for irrelevant operations when a bump is needed.

So the rule is, re-use the same timeout when the operation is similar. In reality though, it's hard to tell which operations are "similar". For example, if I'm waiting for cert manager to create a secret and I do it 5 minutes before I install cert-manager, that obviously is going to take 5+ minutes. If I do the same operation (wait for the secret) but after cert-manager is already installed, it may take only some seconds. Is that the same operation then?